.travis.yml

```diff
@@ -7,6 +7,8 @@ python:
 install:
   - python setup.py install
 script:
-  - python setup.py validate
+  - travis_retry python setup.py validate
+  # TODO: Enable this
```

Comment:
I haven't been able to reproduce the failures that happen only on Travis yet. I may hold off working on this for now.